### ISSUE TITLE
Win: vue.exe file still has java dependency

### ISSUE THREAD

**Issue by rcrath** [digiah/VUE-solutions#3]
the included vue.exe is supposed to be self contained but requires the presence of a java runtime env already installed.

**Comment by pwilkins** [comment]
I don't believe that the VUE executable ever ran without a Java runtime environment installed, though decades have passed since I worked on the VUE project.  I believe that this is a Java "feature" to allow the compiled executable to be independent of the operating system it runs on.  I know that there are popular Java applications that include a JRE as part of their executable distribution, but correcting this in VUE would be a feature request and not a bug.
